apps/browser/settings/searchenginemodel.cpp

```diff
@@ -102,6 +115,10 @@ void SearchEngineModel::install(const QString& title)
 
```

Comment:
QModelIndex modelIndex = index(i, 0);
                    emit dataChanged(modelIndex, modelIndex, QVector<int>() << InstalledRole);

Small fix for this.